cmd/monitor/collector/bootstrap-agent.yaml

```diff
@@ -21,15 +21,33 @@ erda.oap.collector.core:
           - "erda.oap.collector.processor.aggregator@mem-percent"
         exporters:
           #- "erda.oap.collector.exporter.stdout"
-          - "erda.oap.collector.exporter.collector"
+          - "erda.oap.collector.exporter.collector@metrics"
+        batch_size: ${METRIC_PROMETHEUS_BATCH_SIZE:1024}
+
+    external_metrics:
+      - receivers:
+          - "erda.oap.collector.receiver.prometheus-remote-write@external_metrics"
+        processors:
+          - "erda.oap.collector.processor.k8s-tagger@prw"
+          - "erda.oap.collector.processor.modifier"
+          - "erda.oap.collector.processor.modifier@addmeta"
+          - "erda.oap.collector.processor.modifier@docker_container_summary"
+          - "erda.oap.collector.processor.modifier@mysql"
+          - "erda.oap.collector.processor.aggregator@cpu-usage"
+          - "erda.oap.collector.processor.aggregator@mem-percent"
+        exporters:
+          - "erda.oap.collector.exporter.collector@external_metrics"
         batch_size: ${METRIC_PROMETHEUS_BATCH_SIZE:1024}
 
 # ************* receivers *************
 #erda.oap.collector.receiver.dummy:
 #  rate: 3s
 #  metric_sample: '{"name":"kubelet_cadvisor","timeUnixNano":1640936985459000000,"relations":null,"attributes":{"container":"manager","host_ip":"10.118.177.94","id":"/kubepods/burstable/pod164ec226-8106-4904-9bcb-0218a9b2b793/8367a8b0993ebdf8883a0ad8be9c3978b04883e56a156a8de563afa467d49dec","image":"sha256:6cd7bc0e0855164e7ff495c6ec9a37cf8657f8170fe97055ffba2c63343bcedd","instance":"virtual-kubelet-cn-hangzhou-k","name":"8367a8b0993ebdf8883a0ad8be9c3978b04883e56a156a8de563afa467d49dec","namespace":"default","pod":"elasticsearch-operator-776689d978-mjdzq","pod_name":"elasticsearch-operator-776689d978-mjdzq"},"dataPoints":{"container_cpu_usage_seconds_total":13995.161470334,"container_memory_max_usage_bytes":273977344}}'
 
-erda.oap.collector.receiver.prometheus-remote-write:
+erda.oap.collector.receiver.prometheus-remote-write@prometheus:
+
+erda.oap.collector.receiver.prometheus-remote-write@external_metrics:
+  remote_write_url: "${EXTERNAL_METRIC_REMOTE_WRITE_URL:/api/v1/external-remote-write}"
```

Comment:
`external-remote-write` is only for metric use or any monitor types?
If it's metric only, I think you should make this path clearer.

Reply:
remote_write_url it's used for prometheus, so is obvious, that is to use for metric

Reply:
adjusted to `/api/v1/external-prometheus-remote-write`